Sources/tuist/tuist.docc/Articles/Tuist/Users/project-structure.md

```diff
@@ -0,0 +1,41 @@
+# Project structure
+
+Learn about the structure of Tuist projects and how to organize them.
+
+## Overview
+
+Although Tuist projects are commonly used to supersede Xcode projects, they are not limited to this use case. Tuist projects are also used to generate other types of projects, such as templates, plugins, and tasks. This document describes the structure of Tuist projects and how to organize them.
+
+### Xcode projects
+
+Xcode projects are **the most common type of project generated by Tuist.** They are used to build apps, frameworks, and libraries among others. Unlike Xcode projects, Tuist projects are defined in Swift, which makes them more flexible and easier to maintain. Tuist projects are also more declarative, which makes them easier to understand and reason about. The following structure shows a typical Tuist project that generates an Xcode project:
+
+> Note: The definition files are referred to as **manifest files** throughout the documentation.
+
+```bash
+Tuist/
+  Config.swift
+  Package.swift
+  ProjectDescriptionHelpers/
+Projects/
+  App/
+    Project.swift
+  Feature/
+    Project.swift
+Workspace.swift
+```
+
+- **Tuist directory:** This directory has two purposes. First, it signals **where the root of the project is**. This allows constructing paths relative to the root of the project, and also also running Tuist commands from any directory within the project. Second, it's the container for the following files:
+  - **Config.swift:** This file contains configuration for Tuist that's shared across all the projects, workspaces, and environments. For example, it can be used to disable automatic generation of schemes, or to define the deployment target of the projects.
+  - **Package.swift:** This file contains Swift Package dependencies for Tuist to integrate them using Xcode projects and targets (like [CocoaPods](https://cococapods)) that are configurable and optimizable. <!-- TODO: Add a link to the dependencies page -->
```

Comment:
Do you plan to address the TODO in this PR?

Reply:
No. I'll resolve the dependencies as I move the other pages. I'll then do a final pass before considering the documentation rewrite complete.